test/integration/singlecluster/controller/core/suite_test.go

```diff
@@ -72,7 +74,11 @@ func managerSetup(ctx context.Context, mgr manager.Manager) {
 
 	controllersCfg := &config.Configuration{}
 	mgr.GetScheme().Default(controllersCfg)
-
+	emptySelector, _ := metav1.LabelSelectorAsSelector(&metav1.LabelSelector{})
+	metrics.SetLocalQueueMetrics(&metrics.LocalQueueMetricsConfig{
+		Enabled:            true,
+		LocalQueueSelector: emptySelector,
+	})
```

Comment:
Could we specify this parameter only when we enable the feature gate during a dedicated test case?
I think we can take the same way as https://github.com/kubernetes-sigs/kueue/blob/da6cdef8962e6908ffe5e177115c115a289d7091/test/integration/singlecluster/controller/jobs/job/job_controller_test.go#L67-L71

Reply:
`EnabledClusterQueueResources` and `QueueVisibility` are enabled here as well. I'm not seeing a simple way to change this configuration rn

Reply:
Nvm. I think I've come up with a workable solution